comp/volunteers/blue-shirts.yaml

```
@@ -1,6 +1,6 @@
-summary: Order additional blueshirts for the competition
+summary: Order volunteer shirts for the competition
 
-priority: minor
+priority: could
```

Being able to identify volunteers can be incredibly helpful

```suggestion
priority: should
```